builder/docker/tinygo/smoke.sh

```diff
@@ -45,7 +45,6 @@ fi
 
 # Check the base image
 assert_file $base/Dockerfile.base $version 'branch \K.+(?= https://github.com/tinygo-org/tinygo.git)'
-assert_file $base/Dockerfile $version 'FROM suborbital/tinygo-base:\K.+(?= as)'
```

Comment:
Decouples the creation of the base image from the tinygo image. These should be built as separate steps, in separate PRs